### ISSUE TITLE
Maintenance not work

### ISSUE THREAD

**Issue by ghost** [popphp/pop-bootstrap#1]
Hello again,

I'm writing the cms based on the latest version of the libraries and as a starting point I took the sample application in bootstrap version 4 but I can not get the maintenance mode to work.

If I try to activate it, give me this message:

`C:\wamp64\www\appv4\app\src\Http\Web\Event/../../../view/maintenance.phtml`

If I try to activate it before opening the browser the error is as follows:

` Notice: Trying to get property 'username' of non-object in C:\wamp64\www\appv4\app\src\Http\Web\Event\Maintenance.php on line 55`
`C:\wamp64\www\appv4\app\src\Http\Web\Event/../../../view/maintenance.phtml`

Attach the screen.

![cattura](https://user-images.githubusercontent.com/14064365/46018937-4bfc5900-c0db-11e8-9906-0ab38b1aae62.PNG)

Can you help me solve?

.....
**NOTE:** The line number is different from your original file because I added in the PHPDOC section the calls to the two exceptions otherwise the PHPStorm inspection keeps my life under control with all its controls.

The code has not been touched and at the moment I have not added anything that could cause errors or conflicts.

**Comment by ghost** [comment]
I found a stratagem, it is not very elegant but it works.

To resolve the error on the '**username'** property, simply change the cycle from

```php
if ($application->services->isAvailable('session')) {
    $sess = $application->services['session'];
    $view->username = $sess->user->username;
 }
```
to
```php
if ($application->services->isAvailable('session')) {
    $sess = $application->services['session'];
    if( ! empty( $sess->user ) ) {
       $view->username = $sess->user->username;
   }
}
```

and add a /.. for looking the proper view. Change the line

```php
$view     = new View(__DIR__ . '/../../../view/maintenance.phtml');
```

to

```php
$view     = new View( __DIR__ . '/../../../../view/maintenance.phtml');
```

**Comment by nicksagona** [comment]
Ok this issue has been patched and pushed live with v4.1.2. Thanks for catching it.
